### ISSUE TITLE
building broken with clojure 1.8 & develop branch

### ISSUE THREAD

**Issue by selfsame** [arcadia-unity/Arcadia#208]

**Comment by nasser** [comment]
Removing the `clojure.core.server` invocation for `RT`'s startup is part of this fix.

**Comment by selfsame** [comment]
RT.cs's clojure.core.server inclusion can be found at https://github.com/clojure/clojure-clr/blob/447e00568b323a9e6083658133c36cf84b441b19/Clojure/Clojure/Lib/RT.cs#L636-L638 upstream

**Comment by selfsame** [comment]
to get a build users need to add a 
```clj
  (:require 
    clojure.core.server
    arcadia.internal.hook-help)
```
somewhere, this could be done in arcadia.core, so as long as you depend on that there's no additional steps

**Comment by nasser** [comment]
Fixed by #243
